.rakeTasks

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
was this added by accident?

Reply:
I was wondering about that, but after reading this, I assumed it was ok.
![image](https://user-images.githubusercontent.com/69522014/95468120-1949f400-0976-11eb-9a9f-cdc18a428672.png)

Reply:
Will get that removed.